src/main/resources/hudson/plugins/git/GitSCM/help-useCaches.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>When checked, Hudson will maintain a clone of the Git repository
+    in use as a reference for each clone of that repository. The clone
+    will be kept on each node and updated before any other clone of the
```

Comment:
Wow. Sorry for that. I will fix the description when I have something meaningful to fix. Just not to waste the CI for this.